core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/AuditEventV4Controller.java

```diff
@@ -42,18 +49,25 @@ public AuditEventV4Responses getAuditEvents(Long workspaceId, String resourceTyp
 
     }
 
+    public List<AutoscaleClusterHistoryResponse> getAutoscalingEvents(String resourceCrn) {
+        return historyController.getHistoryByCrn(resourceCrn, MAX_AUTOSCALING_EVENTS);
```

Comment:
This call will not work. You shall instead rely on the respective autoscale client and endpoint. Remember, you are about to make a REST call between two microservices!